dxr/query.py

```diff
@@ -313,12 +307,18 @@ def visit_not_term(self, not_term, (not_, term_dict)):
         term_dict['not'] = True
         return term_dict
 
-    def visit_filtered_term(self, filtered_term, (plus, filter, colon, term_dict)):
+    def visit_filtered_term(self, filtered_term, (plus, filter, colon, (term_dict,))):
         """Add fully-qualified indicator and the filter name to the term_dict."""
         term_dict['qualified'] = plus.text == '+'
         term_dict['name'] = filter.text
         return term_dict
 
+    def visit_cased_text(self, cased_text, (at_, term_dict)):
+        """Flip case_sensitive bit in the term_dict."""
+        # Negate our case sensitivity guess if we prepend an @.
+        term_dict['case_sensitive'] = not term_dict['case_sensitive']
```

Comment:
It's strictly less powerful but (a few of us think) quite a bit easier to keep in your head if we make @ mean "case sensitive" rather than "the opposite of what I guessed". What do you think?

(We could add a "case insensitive" operator later if necessary and/or some UI to ignore case on various terms.)